src/Document/Type/Html.php

```diff
@@ -431,12 +431,14 @@ public function countMenuChildren()
 			{
 				$dbo = \App::get('db');
 
-				$query = $dbo->getQuery(true);
-				$query->select('COUNT(*)');
-				$query->from('#__menu');
-				$query->where('parent_id = ' . $active->id);
-				$query->where('published = 1');
-
+				$query = $dbo->getQuery();
```

Comment:
The query starts:

```
$query
    ->select('id', null, true)
    ...
```

The TRUE at the end there tells it to do a COUNT on the field.